internal/providers/gitlab/properties_test.go

```diff
@@ -390,11 +390,7 @@ func TestPropertiesToProtoMessage(t *testing.T) {
 
 // MustNewProperties creates Properties from a map or panics
 func MustNewProperties(props map[string]any) *properties.Properties {
```

Comment:
Do you still want this method?  (If you did, I'd suggest moving it to `pkg/entities/properties.go`, and using it in all the tests, at a minimum.)